docs/config/examples.rst

```diff
@@ -147,6 +147,35 @@ in the UMF repository.
 
 TODO
 
+CTL statistics example
```

Comment:
hmm and here you have the other example explained...? perhaps describe both of them in both places...? (web docs and example's README)